### DISCUSSION TITLE
I cant access to Icons from listTile

### DISCUSSION THREAD

**Discussion by zox47** [flet-dev/flet#3715] [Q&A]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the bug

I cant access to all icons from listtile
![Screenshot 2024-07-25 081849](https://github.com/user-attachments/assets/e1676c74-4469-4726-b615-d5146dd618f4)

### Code sample

listtile.controls.append(ListTile(

                    enable_feedback=True,

                    selected_color=colors.ON_BACKGROUND,
                    trailing=IconsButton(icons.EYE),
               #     shape=RoundedRectangleBorder(radius=100),

                    leading=CircleAvatar(
                    content=Text(first_and_last_letters(message["sender"]), color=colors.ON_BACKGROUND),
                    color=colors.INDIGO),
                    bgcolor=colors.BACKGROUND,))

### To reproduce

![Screenshot 2024-07-25 081849](https://github.com/user-attachments/assets/468d6686-4d5d-4f12-b96f-26d11fc21b01)

### Screenshots / Videos

<details open>
<summary>Screenshots / Video demonstration</summary>

[Upload media here]

</details>

### Operating System

macOS

### Operating system details

Windows 10

### Flet version

0.23.2

### Regression

I'm not sure / I don't know

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

**Comment by ndonkoHenri** [comment]
It says `icons` is a "str" object, meaning you set icons to a string in your code. Search your code for that area.
Share the full code here if you need further assistance.

**Reply by zox47** [reply]
No icons its inside a controle of IconButton and its still said "str"

![Untitled](https://github.com/user-attachments/assets/77a07635-f2e3-470b-8cb1-3593ec49eb40)

**Reply by zox47** [reply]
the only solution here is use cupertino_icons

**Reply by ndonkoHenri** [reply]
Try a simple app with just IconButton and you will certainly see that this error wont occur.
https://flet.dev/docs/controls/iconbutton
